templates/traefik.yaml

```diff
@@ -10,8 +10,8 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "1.68.4-2"
-    appversion.kubeaddons.mesosphere.io/traefik: "1.68.4"
```

Comment:
this appversion points to the chart version, is that right or should it be the version of traefik we're using? (`1.7.12`)

Reply:
should be the version of traefik.

Reply:
thx! fixed. I'll make a second pass tomorrow, I think I saw a couple others that referred to the chart instead of app version